bitnami/redis/templates/sentinel/statefulset.yaml

```diff
@@ -386,7 +386,7 @@ spec:
             {{- if .Values.auth.enabled }}
             {{- if .Values.auth.usePasswordFiles }}
             - name: REDIS_PASSWORD_FILE
-              value: {{ printf "/opt/bitnami/redis/secrets/%s" (include "redis.secretPasswordKey" .) }}
+              value: "/opt/bitnami/redis/secrets/redis-password"
```

Comment:
Thank you very much for your contribution!
I noticed this line also required the fix so I added a commit implementing it.